roles/common/handlers/main.yml

```diff
@@ -36,3 +36,8 @@
 
 - name: Restart Netdata
   service: name=netdata state=restarted
+
+- name: source global_bashrc
+  shell:
+    cmd: source /etc/bash.bashrc
+    executable: /bin/bash
```

Comment:
This isn't needed, it would only apply to the immediate command run which would immediately exit.